Update new-full-time-swe-onboarding-materials.md (#368)
civiform/docs: docs/contributor-guide/developer-guide/new-full-time-swe-onboarding-materials.md

Ensure your team lead grants you access to the following resources.

* Slack
* Added to the civiform-engineers and civiform-technical Google groups
* Github Access
  * [Civiform Org](https://github.com/civiform)
  * civiform/civiform
  * civiform/docs
* DockerHub (optional)

>   * You probably don't need this, as images are pushed automatically by an Exygy account and a DockerHub account is not required in order to use the images, but contact an Exygy engineer if you do at some point.

Reviewer: ```suggestion
  * A DockerHub account is not required to use CiviForm images, so you probably don't need this. Contact an Exygy engineer if you do at some point.
```

* Deployment (optional)
  * AWS

>   * Azure (not currently in use)

Reviewer: suggestion (non-blocking): maybe just delete this? I'm working on deprecating all the Azure things now.

* Authentication (optional)

>   * Auth0 - you can create your own account to use to log in to staging as an applicant

Reviewer: question (non-blocking): I'm confused about this sentence. Is this saying what you would use Auth0 for? I don't think devs need Auth0 to create an account, they can just do that via the CiviForm UI? I only needed Auth0 access when I was setting up our staging instance to add the client id and secret, but I imagine folks might need it if they are working on auth stuff generally as well? I'm not sure.
